process/templates/WG_NAME_graduation_stage.md

```diff
@@ -1,11 +1,41 @@
 ## Working Group graduation application
 
+### WG has met all Incubating requirements
+  * "link to Incubating PR"
+
+### List WG Chair(s) and or Vice Chair
+The WG must have a minimum of 2 Chair(s) or 1 Chair and 1 Vice Chair
+
+  * "name, affiliation, GitHUb ID"
+
 ### List of regular contributors
 The WG must have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
-
   * "name, affiliation, GitHub ID"
 
-### Working Group activity
-The WG must have met at least 4 times over a period of at least 2 months since becoming `Incubating`
+### Release cadence
+The TI must be able to show a consistent release cadence.
+  * "link to release documentation"
+
+### Governance
+Projects must have documented project governance and be able to demonstrate that governance in action.
+  * "link to governance documents"
+Projects have met at least 4 times over a period of at least 2 months since becoming incubating
+  * "link to meeting agenda"
+Have a define roadmap and annual goals for the project
+  * "link to roadmap and goals"
+
+### To retain Graduated status
+A TI is expected to continue operating per the above guidelines, and to provide the TAC with quarterly status updates, and to approach the TAC when seeking approval of substantial changes (such as when accepting or promoting new Projects).
```

Comment:
```suggestion
```

This is true, but is covered by https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md and doesn't need to be part of the application process (and we don't have a section like this on the working group sandbox or incubating template).

Reply:
@SecurityCRob are you still thinking about this change? Or do we want to leave it in?

Reply:
From https://github.com/ossf/tac/pull/279#discussion_r1523586318 it sounds like we'd like to keep these.